data/discord/messages/homeassistant.yaml

```diff
@@ -1334,6 +1334,8 @@ python:
 check:
   description: "Check configuration"
   content: >-
+    To check your configuration use the command for your install method:
+    
     * HAOS [`ha core
```

Comment:
We could use `fields` for the 3 different types to make the formatting better here.